docs/robots/ridgeback/user_manual_ridgeback.mdx

```diff
@@ -1,12 +1,330 @@
+
+To get Ridgeback connected to your local Wi-Fi network, you must first access the Onboard Computer using a wired connection to the robot.
+Open the User Bay, then connect to one of the network ports with a standard Ethernet cable.
+Then, see [Ridgeback Tutorials](/docs/robots/ridgeback/tutorials_ridgeback#ridgeback-networking) for details
+on completing the networking setup.
+
+### Connecting and Using the Controller {#controller}
+
+To use your controller, it must be paired with your Ridgeback and powered on.
+For a standard Ridgeback with the Onboard Computer from Clearpath Robotics, the controller is already paired with your Ridgeback.
+If your controller has become unpaired or if you wish to pair a new controller, following the instructions in the
+[Ridgeback Tutorials](/docs/robots/ridgeback/tutorials_ridgeback#pairing-the-controller).
+
+To teleoperate Ridgeback, first clear the motion-stop.
+Then, power on the controller by pressing the `PS` button on the controller.
+Once the blue LED on the top of the controller goes solid, you are paired and ready to drive.
+
+Hold the L1 trigger button (deadman switch) and push the left thumbstick forward to drive the Ridgeback.
+For full speed mode, hold the R1 trigger.
+See the figure below for the Sony PS4 controls layout.
+If you are not seeing any action, refer to [Support](#ridgeback-support) to get in touch with support.
+
+<center>
+  <figure>
+    <img
+      src="/img/robot_images/ridgeback_images/ridgeback-sony-ps4-controller-labelled.png"
+      width="600"
+    />
+    <figcaption>PS4 Controls Layout</figcaption>
+  </figure>
+</center>
+
+### Using ROS
+
+<ComponentUsingRos />
+
+### Battery Charging
+
+Ridgeback uses two 8A31DTM Group-31 lead-acid (PbSO<sub>4</sub>) deep-cycle AGM batteries mounted low in the
+chassis. They provide an output of 24 VDC (nominal) and 100 Ah of capacity.
+
+When finished with the Ridgeback, press and release the power button to power it off.
+
+Ridgeback has an internal charger. All that is required to charge Ridgeback is to open the User Bay access
+panel, locate the charger power cord and plug it into any 85-270 VAC 50/60Hz mains outlet. The robot can be
+turned "on", though we recommend against driving the robot when the battery is charging.
+
+The battery pack is rugged and designed for the environments into which Ridgeback may be deployed.
+However, please take note of the following:
+
+- The batteries and robot must not be stored or operated above 50 ◦C or below -20 ◦C.
+- If the robot is to be stored for any length of time it should be in a cool, dry location. The batteries should
+  be fully charged and periodically maintained at full charge to ensure a long service life. Ridgeback
+  should not be stored with a low state of charge nor for extended periods of time without top-up charging.
+  The batteries may suffer from "sulfation" which reduces their capacity and lifetime.
+- Do not allow the batteries to reach a depth of discharge of 80.
```

Comment:
80% ? 
Are we suggesting the User does not allow the robot to reach a state-of-charge below 20%?

Reply:
That was carried forward from the old User Manual. I'll confirm with JB.

Reply:
I confirmed that number with JB. Added the missing '%'.